qiskit/pulse/model/pulse_target.py

```diff
@@ -20,15 +20,80 @@
 from qiskit.pulse.exceptions import PulseError
 
 
-class LogicalElement(ABC):
+class PulseTarget(ABC):
+    """Base class of pulse target.
+
+    When playing a pulse on a quantum hardware, one typically has to define on what hardware component
+    the pulse will be played, and the frame (frequency and phase) of the carrier wave.
+    :class:`PulseTarget` addresses the first of two, and identifies the component which is the target
+    of the pulse. Every played pulse and most other instructions are associated with a
+    :class:`PulseTarget` on which they are performed.
+    """
+
+    def __init__(self, hash_identifier):
+        """Create ``PulseTarget``.
+
+        Args:
+            hash_identifier: A hashable unique identifier.
+        """
+        self._hash = hash((hash_identifier, type(self)))
+
+    def __hash__(self) -> int:
+        return self._hash
+
+
+class Port(PulseTarget):
+    """A ``Port`` type ``PulseTarget``.
+
+    A :class:`Port` is the most basic ``PulseTarget`` - simply a hardware port responsible for
+    pulse emission. A :class:`Port` is identified by a string, which must be recognized by the
```

Comment:
```suggestion
    A :class:`Port` is the most basic ``PulseTarget`` - simply a hardware port responsible for
    pulse emission. A :class:`Port` is identified by a string, which must be recognized by the
```
We don't have port for signal acquisition?

Reply:
Good point. Thanks.